cognite/client/_api/geospatial.py

```diff
@@ -0,0 +1,531 @@
+        )
+
+    def search_features(
+        self,
+        feature_type: FeatureType,
+        filter: Dict[str, Any],
+        properties: Dict[str, Any] = None,
+        limit: int = 100,
+        order_by: List[OrderSpec] = None,
+        allow_crs_transformation: bool = False,
+    ) -> FeatureList:
+        """`Search for features`
+        <https://docs.cognite.com/api/v1/#operation/searchFeatures>
+
+        Args:
+            feature_type: the feature type to search for
+            filter (Dict[str, Any]): the search filter
+            limit (int): maximum number of results
+            properties (Dict[str, Any]): the output property selection
+            order_by (List[OrderSpec]): the order specification
+            allow_crs_transformation: If true, then input geometries will be transformed into the Coordinate Reference
+                System defined in the feature type specification. When it is false, then requests with geometries in
+                Coordinate Reference System different from the ones defined in the feature type will result in
+                CogniteAPIError exception.
+
+        Returns:
+            FeatureList: the filtered features
+
+        Examples:
+
+            Search for features:
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> my_feature_type = c.geospatial.retrieve_feature_types(external_id="my_feature_type")
+                >>> my_feature = c.geospatial.create_features(my_feature_type, Feature(external_id="my_feature", temperature=12.4))
+                >>> res = c.geospatial.search_features(my_feature_type, filter={"range": {"property": "temperature", "gt": 12.0}})
+                >>> for f in res:
+                ...     # do something with the features
+
+            Search for features and select output properties:
+
+                >>> res = c.geospatial.search_features(my_feature_type, filter={}, properties={"temperature": {}, "pressure": {}})
+
+            Search for features and order results:
+
+                >>> res = c.geospatial.search_features(my_feature_type, filter={}, order_by=[OrderSpec("temperature", "ASC"), OrderSpec("pressure", "DESC")])
+
+        """
+        resource_path = self._feature_resource_path(feature_type) + "/search"
+        cls = FeatureList
+        order = None if order_by is None else [f"{item.property}:{item.direction}" for item in order_by]
+        res = self._post(
+            url_path=resource_path,
+            json={
+                "filter": filter,
+                "limit": limit,
+                "output": {"properties": properties},
+                "sort": order,
+                "allowCrsTransformation": (True if allow_crs_transformation else None),
+            },
+        )
+        return cls._load(res.json()["items"], cognite_client=self._cognite_client)
+
+    def stream_features(
+        self,
+        feature_type: FeatureType,
```

Comment:
```suggestion
        feature_type_external_id: str,
```

Reply:
I don't understand your suggestion

Reply:
https://github.com/cognitedata/cognite-sdk-python/pull/873#discussion_r784798189

Reply:
weird, github had decided to hide your initial comment...
should be good now